api/src/main/java/org/accula/api/handlers/response/User.java

```diff
@@ -0,0 +1,19 @@
+package org.accula.api.handlers.response;
+
+import lombok.Value;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * @author Anton Lamtev
+ */
+@Value
+public class User implements ResponseBody {
+    Long id;
+    String login;
+    String name;
+
+    public static User from(final org.accula.api.db.dto.User user) {
```

Comment:
As far as I know DTO objects are made for representation layer of the application. But here you map dto.User object into another User Object

Reply:
You may have a look at [MapStruct](https://mapstruct.org/documentation/stable/reference/html/) - it does help with conversion between entity and dto objects

Reply:
I totally agree with you! We have to use MapStruct in the future. And __dto__ package name is not the best one. So we have to rename it next commit which will modify these files.